packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -96,6 +97,11 @@ export function serialize(message: PubsubMessage): Uint8Array {
         doc: message.stream.toString(),
         stream: message.stream.toString(),
         tip: message.tip.toString(),
+        /*
+          @active-branch
+          placeholder, optional type
+        */
+        model: message.model?.toString()
```

Comment:
what happens if model is missing?  Then the `payload` object will have `model: undefined`.  How will `textEncoder.encode()` handle that?  Will it strip out the `model` field (desired behavior), or will it still include the `model` field as an explicit `undefined` (not desired behavior as it increases message size unnecessarily)?

Reply:
instead of trying to figure out what `textEncoder` does here, it may be better to just proactively only add the `model` field to the payload if you know it's non-null

Reply:
looks like this is back to always adding `model` instead of only adding it if it's not undefined

Reply:
I don't think this matters since `JSON.stringify` (on line 108) removes all properties that are not defined.

Reply:
`undefined` is not a thing in JSON.

Reply:
Okay cool, in that case you can probably just delete line 107 and call it a day